examples/rembi-metadata.yaml

```diff
@@ -37,6 +39,7 @@ biosamples:
       ncbi_taxon: NCBI:txid10090
     biological_entity: mouse embryo volumes
     description: "mouse embryo volumes which were acquired in utero and in vivo from pregnant mice (10-14.5 days after mating)"
+    intrinsic_variables: [Homozygous GFP integration into mitotic genes]
```

Comment:
yaml nitpick, but the []'s don't appear in the converted output. Should this be quoted, or the []'s removed if not relevant?

Reply:
They shouldn't appear in the converted output (pagetab). But they have to be in [] in the yaml as the expected value is a list in BIA REMBI model.

Reply:
oh in this case can it be

```yaml
intrinsic_variables:
  - Homozygous GFP integration into mitotic genes
```

Just to keep consistency with ususal yaml syntax?